### PR TITLE
feat: add qr code

### PR DESCRIPTION
**Pull request by rafaelramalho19**
Closes https://github.com/ipfs-shipyard/ipfs-share-files/issues/78

Example: 
image

### CODE REVIEW

File: src/components/copy-link/CopyLink.js [hunk 1]
```diff
@@ -42,6 +43,12 @@ class CopyLink extends React.Component {
             </div>
           </CopyToClipboard>
         </div>
+        <div className="overflow-hidden">
+          <div className="flex flex-column items-center appear-from-below">
+            <span className="charcoal f5 pb2">{t('copyLink.qrLabel')}</span>
```
Comment (lidel) [inline src/components/copy-link/CopyLink.js:48]: I suspect we want to make font size smaller for this label, perhaps the same as "This link changes.." ?

>

**Review by lidel — APPROVED** [review]
Scanned QR code with Firefox for Android and it workeeeed :rocket: 

Small nit inline.

File: src/components/copy-link/CopyLink.js (hunk 1 above)
Comment (jessicaschilling) [reply src/components/copy-link/CopyLink.js:48]: @rafaelramalho19 just put that there so I could style it. Watch this space.

**Comment by jessicaschilling** [comment]
Note: discovered/discussed with @rafaelramalho19 some display weirdness for long i18n translations of "Add file" - including tweak https://github.com/ipfs-shipyard/ipfs-share-files/pull/102/commits/d7584df6d511405f2618cb8d02535e42aba89ebd for that in this PR since we're in the same geographical area.

**Comment by jessicaschilling** [comment]
Also confirmed QR code works in iOS native photos app.

**Comment by jessicaschilling** [comment]
Updated screenshot ... visual tweaks.
image

**Review by jessicaschilling — APPROVED** [review]

**Pull request merged**